drivers/gles2/rasterizer_scene_gles2.cpp

```diff
@@ -2991,7 +3001,7 @@ void RasterizerSceneGLES2::render_scene(const Transform &p_cam_transform, const
 		//copy screen texture
 
 		if (storage->frame.current_rt->multisample_active) {
-			// Resolve framebuffer to front buffer before copying
+		// Resolve framebuffer to front buffer before copying
```

Comment:
@akien-mga any idea why clang-format is flipping this and later on line 3063 as well? I'll fix it up when I have time to work on this next.

Reply:
clang-format was always a bit difficult around preprocessor defines, but in recent versions it seems to work ok.

What version do you use?

Reply:
Still on 5.0.0 on Windows

Reply:
That's too old, and indeed one of the versions which was buggy around `#define` and `#ifdef`.

We require 8.0.0 or later now: http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#using-clang-format-locally

Reply:
Downloading 5.0.2, or should I go to a higher version?

Reply:
ok 8 it is :)

Reply:
Too old too, you need 8.0.0 or later.

Reply:
Same issue with 8. When it's wrong it will correct it, when its correct it'll make it wrong. But atleast with a commit I can override it, it wouldn't let me when cherry-picking :)